README.md

```diff
@@ -178,10 +178,10 @@ We hope you will enjoy this application and that it will help you to easily orga
 
 
 <a name="sec_4"></a>
-## Developers
+## 4. Developers
 * Roxane Burri
 * Victor Carles
 * Anthony Guinchard
 * Jean-Luc Stoupy
 * Mark Tropin
-* Kepler Warrington-Arroyo
\ No newline at end of file
+* Kepler Warrington-Arroyo
```

Comment:
You forget Kepler ?

Reply:
I don't remember to have forgotten him ^^'